README.md

```diff
@@ -142,18 +119,10 @@ Facilitators:
 - Sara Trapetti ([@SaraTrap](https://github.com/SaraTrap)), SIGHUP
 - Jade Applegate ([@jadeapplegate](https://github.com/jadeapplegate)), Indeed
 
-### Mentoring WG
-
-[Charter](/mentoring/README.md)
-
-Facilitators:
-
-- Nate Waddington ([@nate-double-u](https://github.com/nate-double-u)), CNCF
-- Jay Tihema ([@jaytiaki](https://github.com/jaytiaki)), ii
-- Ihor Dvoretskyi ([@idvoretskyi](https://github.com/idvoretskyi)), CNCF
-
 ### Community Infrastructure Team
 
+Status: [Proposed](https://hackmd.io/mmPVK1PCQD2swm1GrzB0NQ)
```

Comment:
Community Infrastructure was suggested in that doc and we agreed to collaborate on the initiative through Contributor Growth, so it's no longer proposed.

How about we move this section for Community Infrastructure to the Contributor Growth readme? And then we can change facilitators to something else, since it was actually a list of interested people as again it's not a formal group with a "lead".

Reply:
Agree. What do you think about the Contributor Growth readme structure.
Do I start a new section under "Documentation" or slot it in somewhere in the middle?

Reply:
How about a top level section before or after Documentation that lists initiatives that we are working to start _and staff_.

Reply:
Good idea, created a new section for intitiatives in the contributor groth structure.